requirements.txt

```diff
@@ -1,43 +1,27 @@
-aniso8601==7.0.0
```

Comment:
How come we have all these removed?

Reply:
I regenerated the requirements.txt file. I don't see any packages missing.

Reply:
Oh, yes. I got confused due to regeneration.